modules/next_stage/php/NDB_Form_next_stage.class.inc

```diff
@@ -34,55 +68,67 @@ class NDB_Form_next_stage extends NDB_Form
         $timePoint->setData("Date_".$newStage, $values['date1']);
 
         // set SubprojectID if applicable
-        if(isset($values['SubprojectID'])) {
+        if (isset($values['SubprojectID'])) {
             $timePoint->setData('SubprojectID', $values['SubprojectID']);
         }
 
         // set scan done if applicable
-        if(isset($values['scan_done'])) {
+        if (isset($values['scan_done'])) {
             $timePoint->setData('Scan_done', $values['scan_done']);
         }
 
         // create a new battery object && new battery
-        $battery = new NDB_BVL_Battery;
-        $candidate =& Candidate::singleton($timePoint->getCandID() );
+        $battery   = new NDB_BVL_Battery;
+        $candidate =& Candidate::singleton($timePoint->getCandID());
 
         $firstVisit = false;
-        $vLabel = $candidate->getFirstVisit();//get first visit for candidate
-        if($vLabel ==  $timePoint->getVisitLabel()){
-            $firstVisit = true; //if current visit label is same as visit label returned must be first visit
+        $vLabel     = $candidate->getFirstVisit();
+         //get first visit for candidate
```

Comment:
I don't think this comment is even necessary. The function call is `$candidate->getFirstVisit()`